phys/module_radiation_driver.F

```diff
@@ -1836,26 +1869,6 @@ SUBROUTINE radiation_driver (                                  &
                 ENDDO
              END IF
 
-             !Need to reset NLAYERS if vertical nesting is used.
-             !This code follows that for case RRTMSCHEME within
-             !subroutine RRTMLWRAD.
-             IF ( PRESENT(p_top) ) THEN
-                p_top_dummy = p_top
-             ELSE
-                p_top_dummy = -1. ! not used by NMM
-             END IF
-             IF ( p_top_dummy .GT. 0 ) THEN ! flag value for NMM = -1
-                !NLAYERS is recalculated
-                !every time the radiation scheme is called. This is
-                !necessary if e_vert parent .NE. e_vert nest since
-                !NLAYERS could then be different for each domain.
-                CALL RRTMG_LWINIT(                             &
-                              p_top, .FALSE. ,                 &
-                              ids, ide, jds, jde, kds, kde,    &
-                              ims, ime, jms, jme, kms, kme,    &
-                              its, ite, jts, jte, kts, kte     )
-             ENDIF
-
              CALL RRTMG_LWRAD(                                      &
```

Comment:
@rmfonsecaweb This section of code cannot be deleted - this is part of the new feature recently introduced to the model.

Reply:
Agree with the reviewer. Please accept our apologies for having unintentionally removed this section of the code. Thank you!

Reply:
We will add again that section of the code to file. Apologies for having unintentionally deleted it when modifying "module_radiation_driver.F" yesterday. Thank you!